life_expectancy/tests/test_cleaning.py

```diff
@@ -1,16 +1,45 @@
 """Tests for the cleaning module"""
```

Comment:
This is a suggestion. It is good practice to separate your tests by modules. So, my suggestion would be to create separate testing modules for each module (data cleaning, data loading/saving and main).

Reply:
Thank you for your suggestion, already changed it!! Can we merge?